src/main/java/seedu/address/ui/AutoCompletePanel.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.layout.Region;
+import seedu.address.autocomplete.AutoCompleteWord;
+import seedu.address.autocomplete.AutoCompleteWordHandler;
+
+/**
+ * Panel containing the list of suggested words.
+ */
+public class AutoCompletePanel extends UiPart<Region> {
+    private static final String FXML = "AutoCompletePanel.fxml";
+
+    private AutoCompleteWordHandler autoCompleteWordHandler;
+
+    @FXML
+    private ListView<AutoCompleteWord> autoCompleteWordListView;
+
+    public AutoCompletePanel(AutoCompleteWordHandler autoCompleteWordHandler) {
+        super(FXML);
+        this.autoCompleteWordHandler = autoCompleteWordHandler;
+        autoCompleteWordListView.setItems(autoCompleteWordHandler.getOListAutoCompleteWords());
+        autoCompleteWordListView.setCellFactory(listView -> new AutoCompleteListViewCell());
+    }
+
+    /**
+     * Custom {@code ListCell} that displays the graphics of a {@code AutoCompleteWord} using a {@code AutoCompleteCard}
+     */
+    class AutoCompleteListViewCell extends ListCell<AutoCompleteWord> {
+        @Override
+        protected void updateItem(AutoCompleteWord autoCompleteWord, boolean empty) {
+            super.updateItem(autoCompleteWord, empty);
+            System.out.println("updateItem called");
```

Comment:
Note System.out.println, may want to use Logger instead? This is more a nitpick though.